src/render.asm

```diff
@@ -1,110 +1,99 @@
+// These functions are in .data because they may be called from both cores.
```

Comment:
My skills regarding writing/reading assembly (arm or otherwise) are not proficient. So I do not have much to comment on this code. I understand how these are called/used, but can not really audit the function of them.